doc/user-guides/auth/authzed.md

```diff
@@ -0,0 +1,345 @@
+# AuthPolicy Integration with Authzed/SpiceDB
+
+This guide explains how to configure permission requests for a Google Zanzibar-based [Authzed/SpiceDB](https://authzed.com) instance using gRPC.
+
+## Prerequisites
+
+You have installed Kuadrant in a [kubernetes](https://docs.kuadrant.io/latest/kuadrant-operator/doc/install/install-kubernetes/) or [OpenShift](https://docs.kuadrant.io/latest/kuadrant-operator/doc/install/install-openshift/) cluster.
```

Comment:
these links have now been updated so they wont work when the new version of the docs is released on the website https://docs.kuadrant.io/dev/kuadrant-operator/doc/install/install-openshift/

Reply:
Seems most other guides were updated to https://github.com/Kuadrant/kuadrant-operator/blob/main/doc/user-guides/ratelimiting/authenticated-rl-for-app-developers.md#setup-the-environment,  is this what we want here instead now or do you have an example? 🤔

Reply:
The setup looks to still use the `make` targets 🤷

Reply:
We have issue to remove the make target altogether for existing docs [Remove/replace using make target as the method of installation for docs. #1093](https://github.com/Kuadrant/kuadrant-operator/issues/1093) that's going to be worked on soon. In this case its the links you have above, have changed in a new version of the website. We now link off to helm. The install-kubernetes doc no longer exists. The openshift doc has been updated but the link is same i just linked to it as a way to get to the dev view of the install guides on the website.

Reply:
Cool 👍 I'll link the setup to either use Helm or if using Openshift 🤔 
A question on this though, is are we expecting this to be cherry picked to the v1 branch? 🤔 
If so, then I guess we should not hard link to the docs website but use relative paths and also not to the dev docs? Or would be cherry pick and have changes to work with the current v1 setup guides ?

Reply:
thats a good question that im not sure i know we are working towards doing a v1.1 release later on in q1 but that seems late enough to make these change live. Yeah relative sounds good to me what you could also do is have similar to what we have for prereq in the SCP doc and just say "Kubernetes cluster with Kuadrant operator installed.". Oh yeah we don't want to use the dev view

Reply:
Ok found more info, if the doc is directly needed by 1.0 like a fix etc then cherry pick if not wait for the next release it will be seen in the dev view